CONTRIBUTORS.md

```diff
@@ -0,0 +1,117 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via an [issue](https://github.com/sigstore/rekor/issues).
+
+## Pull Request Process
+
+1. Create an [issue](https://github.com/sigstore/rekor/issues)
```

Comment:
should we make this more generic so we don't end up with all issues with in rekor?

Reply:
yep, good call. this was just a lazy copy over from rekor, I will make it specific to this repo.